plugins/announcements-velocity/src/main/resources/config.yml

```diff
@@ -0,0 +1,10 @@
+
+# messages must be in json format
+# https://minecraft.tools/en/json_text.php
+scheduledAnnouncements:
+  - cron: "45 1 * * *"
+    message: "{\"text\":\"Restart in 15 minutes\",\"color\":\"gold\"}"
```

Comment:
is it possible to support minimessage here?

Reply:
i guess? but id probably do it in another pr if your okay with that

Reply:
would rather it be in one pr